tests/test-cases/Group24-Host-Affinity/24-01-Basic.md

```diff
@@ -0,0 +1,40 @@
+Suite 24-01 - Basic
+===================
+
+# Purpose:
+To verify basic VM-Host Affinity functionality
+
+# References:
+1. [The design document](../../../doc/design/host-affinity.md)
+
+# Environment:
+This test requires a vCenter Server environment where VCHs can be deployed and container VMs created.
```

Comment:
How does this fare on a ROBO system without DRS? Or on a cluster with DRS disabled?
Talk to @cgtexmex for how best to confirm.

Reply:
Groups / Rules can be created, but since DRS is disabled or not licensed the grouping is not utilized.  The requirement then would of course be DRS Enabled clusters -- which is the requirement currently and testing of DRS Disabled will most likely occur in nightlies to begin and then move into _standard_ CI  later

Reply:
Updated the document with a note explaining that because these tests just look at group management (and not utilization of the group by DRS), this suite should work fine even if DRS is disabled.